### ISSUE TITLE
Access to form.busy

### ISSUE THREAD

**Issue by morloderex** [cklmercer/vue-laravel-forms#1]
Hello @cklmercer 

I recently found this new project you have created here on github through a friend of mine. 

But as i look through the source code i cannot see any way to get access to form.busy like laravel spark allows us to.

Can you impliment that?

**Comment by cklmercer** [comment]
It should already exist.

Assuming you have one of the following two components..

```
Vue.component('example', {
    data() {
        return { 
            form: new AppForm({
                'field': ''
            })
        }
    }
}
```

```
Vue.component('example', {
    forms: {
        form: {
            field: ''
        }
    }
}
```

You can access busy using

```
vm.form.busy
```

If you would like to see where it exists in the source, check `src/form.js:10`.

**Comment by cklmercer** [comment]
Ah, so I wasn't creating new Form objects with forms created from the `forms` option. 

This should be fixed in versions `1.0.7-beta`+
